src/LiveComponent/doc/index.rst

```diff
@@ -3851,6 +3851,23 @@ uses Symfony's test client to render and make requests to your components::
             // Assert that an event was not emitted
             $this->assertComponentNotEmitEvent($testComponent->render(), 'decreaseEvent');
 
+            // dispatch events
+            $testComponent
+                ->dispatchBrowserEvent('browserEvent')
+                ->dispatchBrowserEvent('browserEvent', ['amount' => 2, 'unit' => 'kg']) // dispatch a browser event with arguments
+            ;
+
+            // Assert that the event was dispatched
+            $this->assertComponentDispatchEvent($testComponent->render(), 'browserEvent')
```

Comment:
Since we use `dispatchBrowserEvent` to dispatch a **browser** event, it makes more sense to re-use "dispatchBrowserEvent" in the method name:
```suggestion
            $this->assertComponentDispatchBrowserEvent($testComponent->render(), 'browserEvent')
```